### DISCUSSION TITLE
syncPermissions

### DISCUSSION THREAD

**Discussion by parsadavarzani** [spatie/laravel-permission#2662] [Q&A] [answered]
I have a problem
when im trying to make a role 
im sending permissions ids to backend and when trying to store them with this code $role->syncPermissions() it give me this error :
There is no permission named `2` for guard `web`.
what is it problem ???
can't i use permissions id to store them for a role ???

**Comment by parallels999** [comment, marked as answer]
- https://github.com/spatie/laravel-permission/issues/2530#issuecomment-1797964498

**Reply by parallels999** [reply]
Well, I thank myself for my quick responses. 😄
